tracing-appender/src/rolling.rs

```diff
@@ -154,7 +153,7 @@ pub fn minutely(
 /// # #[clippy::allow(needless_doctest_main)]
 /// fn main () {
 /// # fn doc() {
-///     let appender = tracing_appender::rolling::hourly("/some/path", "rolling.log");
+///     let appender = tracing_appender::rolling::hourly("/some/path", "rolling.log").unwrap();
 ///     let (non_blocking_appender, _guard) = tracing_appender::non_blocking(appender);
 ///
 ///     let collector = tracing_subscriber::fmt().with_writer(non_blocking_appender);
```

Comment:
Would it be better to change the doc function signature to return a `io::Result<()>` and then use `let appender = tracing_appender::rolling::hourly("/some/path", "rolling.log")?;` instead?
Or is it ok with just the `unwrap` for the example?

Reply:
Either is fine --- if we're going to panic in the example, I might prefer an `expect` with a message, so that the reader can see what cases would result in an error being returned. I would be fine with an `expect`, or with `?` --- whatever you think is best!